### PR TITLE
DE-262 Definition of base URL and .env files

### PR DESCRIPTION
**Pull request by paangaflo**
Feat: [DE-262](https://makingsense.atlassian.net/browse/DE-262)

### CODE REVIEW

File: .env.production [hunk 1]
```diff
@@ -0,0 +1,2 @@
+INLINE_RUNTIME_CHUNK=false
```
Comment (andresmoschini) [inline .env.production:1]: What is this configuration?

File: .env.test [hunk 2]
```diff
@@ -0,0 +1 @@
+CI=true
```
Comment (andresmoschini) [inline .env.test:1]: how is this run? in what environment?

File: .env.production [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+PUBLIC_URL=https//cdn.fromdoppler.com/unlayer-editor
```
Comment (andresmoschini) [inline .env.production:2]:
```suggestion
PUBLIC_URL=https://cdn.fromdoppler.com/unlayer-editor
```

File: .env.production (hunk 1 above)
Comment (andresmoschini) [reply .env.production:1]: It generates a file for the runtime in place of including it in the index.html

File: .env.test (hunk 2 above)
Comment (andresmoschini) [reply .env.test:1]: It depends on the command. Build uses `production`, etc.

**Review by maurocardinali — APPROVED** [review]
🚀

File: .env.test (hunk 2 above)
Comment (paangaflo) [reply .env.test]: Removed file.

File: .env.production (hunk 3 above)
Comment (paangaflo) [reply .env.production]: Changed.

File: .env.production (hunk 1 above)
Comment (paangaflo) [reply .env.production:1]: Correct.

**Pull request merged**